Dockerfile

```diff
@@ -3,8 +3,7 @@ FROM fedora:20
 ADD tomprince-hybridlogic-fedora-20-x86_64.repo /etc/yum.repos.d/
 #RUN yum upgrade -y
 RUN yum install -y python-devel python-pip gcc libffi-devel openssl-devel
-RUN pip install buildbot==0.8.9 txgithub boto service_identity
-
+RUN pip install buildbot==0.8.10 txgithub boto service_identity
```

Comment:
Is this due to "Fixed master-side source steps to respect the specified timeout when removing files" in the buildbot changelog?
 * http://docs.buildbot.net/current/relnotes/index.html

Or some other reason?